node/pkg/websocketfetcher/providers/kucoin/kucoin.go

```diff
@@ -29,14 +29,16 @@ func New(ctx context.Context, opts ...common.FetcherOption) (common.FetcherInter
 	fetcher.FeedDataBuffer = config.FeedDataBuffer
 
 	symbols := []string{}
+
 	for feed := range fetcher.FeedMap {
 		symbols = append(symbols, feed)
 	}
 
+	// subscribe to market snapshot to avoid potential rate limit
```

Comment:
@nick-bisonai Can you please expand on how it can avoid potential rate limit? Do you mean that different endpoints have different rate limits, and this one has higher rate limit than `/marker/ticker`?

Reply:
it's a wrong comment, I'll remove it, I'm not using market snapshot anymore